VenafiPS/Public/New-TppCustomField.ps1

```diff
@@ -0,0 +1,214 @@
+<#
+.SYNOPSIS
+Create new custom field in Venafi of the specified class and type.
+
+.DESCRIPTION
+Creates a new custom field in Venafi. Required parameters are name, label, class and type.
+See Venafi docs for information about other metadata items.
+
+.PARAMETER Name
+Name of new custom field
+
+.PARAMETER Label
+Label of new custom field
+
+.PARAMETER Class
+Class of new custom field, either 'Device' or 'X509 Certificate'
+
+.PARAMETER Type
+Type of new custom field
+
+.PARAMETER VenafiSession
+Session object created from New-VenafiSession method.  The value defaults to the script session object $VenafiSession.
+
+.INPUTS
+None
+
+.OUTPUTS
+Query returns a PSCustomObject with the following properties:
+    AllowedValues
+    Class
+    DateOnly
+    DN
+    ErrorMessage
+    Guid
+    Help
+    Label
+    Mandatory
+    Name
+    Policyable
+    RegularExpression
+    RenderHidden
+    RenderReadOnly
+    Single
+    TimeOnly
+    Type
+
+.EXAMPLE
+New-VenafiCustomField -Name "Last Date" -Label "Last Date" -Class "X509 Certificate" -Type "1" -RenderReadOnly -Help "Last Date of certificate import"
+Create new custom field for certificates
+
+.LINK
+http://VenafiPS.readthedocs.io/en/latest/functions/New-TppCustomField/
+
+.LINK
+https://docs.venafi.com/Docs/20.4SDK/TopNav/Content/SDK/WebSDK/r-SDK-POST-Metadata-DefineItem.php?tocpath=Web%20SDK%7CMetadata%20custom%20fields%20programming%20interface%7C_____4
+
+.LINK
+https://docs.venafi.com/Docs/20.4SDK/TopNav/Content/SDK/WebSDK/r-SDK-POST-Metadata-Item.php
+
+#>
+
+function New-TppCustomField {
```

Comment:
need to add function to FunctionsToExport in VenafiPS.psd1

Reply:
Added to FunctionsToExport